### ISSUE TITLE
xmlns and src attributes of the even-numbered *.svg source aren't removed

### ISSUE THREAD

**Issue by ark-tds** [popeindustries/inline-source#80]
version: 6.1.8

---

INPUT: src/html/index.html

```index.html
<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="UTF-8">
  <title></title>
</head>
<body>
  <img src="../img/right.svg" alt="" inline>
  <img src="../img/right.svg" alt="" inline>
  <img src="../img/right.svg" alt="" inline>
  <img src="../img/right.svg" alt="" inline>
  <img src="../img/left.svg" alt="" inline>
  <img src="../img/left.svg" alt="" inline>
  <img src="../img/left.svg" alt="" inline>
  <img src="../img/left.svg" alt="" inline>
</body>
</html>
```

src/img/right.svg

```right.svg
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 32 32"><rect x="6" y="15" width="15" height="2"/><polygon points="20 20 20 12 25 16 20 20"/></svg>
```

src/img/left.svg
```left.svg
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 32 32"><rect x="11" y="15" width="15" height="2" transform="translate(37 32) rotate(180)"/><polygon points="12 12 12 20 7 16 12 12"/></svg>
```

index.js
```index.js
const { inlineSource } = require('inline-source');
const fs = require('fs');

inlineSource('src/html/index.html')
.then(html => {
  fs.writeFileSync('dist/html/index.html', html);
}).catch(err => {
  console.log('failed');
});
```

OUTPUT: dist/html/index.html
```index.html
<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="UTF-8">
  <title></title>
</head>
<body>
  <svg viewBox="0 0 32 32"><path d="M6 15h15v2H6z"/><path d="M20 20v-8l5 4-5 4z"/></svg>
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 32 32" src="../img/right.svg"><path d="M6 15h15v2H6z"/><path d="M20 20v-8l5 4-5 4z"/></svg>
  <svg viewBox="0 0 32 32"><path d="M6 15h15v2H6z"/><path d="M20 20v-8l5 4-5 4z"/></svg>
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 32 32" src="../img/right.svg"><path d="M6 15h15v2H6z"/><path d="M20 20v-8l5 4-5 4z"/></svg>
  <svg viewBox="0 0 32 32"><path d="M26 17H11v-2h15z"/><path d="M12 12v8l-5-4 5-4z"/></svg>
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 32 32" src="../img/left.svg"><path d="M26 17H11v-2h15z"/><path d="M12 12v8l-5-4 5-4z"/></svg>
  <svg viewBox="0 0 32 32"><path d="M26 17H11v-2h15z"/><path d="M12 12v8l-5-4 5-4z"/></svg>
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 32 32" src="../img/left.svg"><path d="M26 17H11v-2h15z"/><path d="M12 12v8l-5-4 5-4z"/></svg>
</body>
</html>
```

EXPECTED: dist/html/index.html
```index.html
<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="UTF-8">
  <title></title>
</head>
<body>
  <svg viewBox="0 0 32 32"><path d="M6 15h15v2H6z"/><path d="M20 20v-8l5 4-5 4z"/></svg>
  <svg viewBox="0 0 32 32"><path d="M6 15h15v2H6z"/><path d="M20 20v-8l5 4-5 4z"/></svg>
  <svg viewBox="0 0 32 32"><path d="M6 15h15v2H6z"/><path d="M20 20v-8l5 4-5 4z"/></svg>
  <svg viewBox="0 0 32 32"><path d="M6 15h15v2H6z"/><path d="M20 20v-8l5 4-5 4z"/></svg>
  <svg viewBox="0 0 32 32"><path d="M26 17H11v-2h15z"/><path d="M12 12v8l-5-4 5-4z"/></svg>
  <svg viewBox="0 0 32 32"><path d="M26 17H11v-2h15z"/><path d="M12 12v8l-5-4 5-4z"/></svg>
  <svg viewBox="0 0 32 32"><path d="M26 17H11v-2h15z"/><path d="M12 12v8l-5-4 5-4z"/></svg>
  <svg viewBox="0 0 32 32"><path d="M26 17H11v-2h15z"/><path d="M12 12v8l-5-4 5-4z"/></svg>
</body>
</html>
```

**Comment by popeindustries** [comment]
Very strange. Unfortunately, I'm unable to reproduce this behaviour locally. Is there any way you could debug this further?

**Comment by vincentorback** [comment]
Can confirm this.
Also for me, every other svg-element that is inlined has the `src`-attribute used on the img-element.

**Comment by popeindustries** [comment]
should be fixed now in `6.2.0`
